### DISCUSSION TITLE
[Enhancement] Features for map

### DISCUSSION THREAD

**Discussion by alexandresanlim** [dotnet/maui#209] [General]
## Summary
The [actual solution](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/map/) have only basic features, I would be interesting if it had functions like custom pins and direction, I know that Xamarin.Essentials has some features to map, but it case launch out the app.

## API Changes
Xamarin.Forms.Map

## Intended Use Case
Use to trace direction from A point to B point.
Use to custom apps with pins.
Use to delivery and ride apps.

**Comment by themronion** [comment]
I think the directions api won't be integrated because services that are used for building routes are mostly not free. Hence, for them to be integrated in Xamarin - someone must pay :) But about custom pins - i agree, i am curious how it isn't implemented yet ( using Xamarin.Forms.GoogleMaps for this currently)
